proxy_init/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:xenial
```

Comment:
🤔 This will make for a large image giving this little functionality. But it's OK if that's what we had before. Could be a future optimization.

Reply:
Agreed, I usually go with Alpine based images, but I didn't want to spent a ton of time debugging if switching was to break things. One of the slim Debian images might be a good choice in the future.